src/maingoblint.ml

```diff
@@ -577,11 +577,7 @@ let do_html_output () =
 
 let do_gobview cilfile =
   let gobview = GobConfig.get_bool "gobview" in
-  let goblint_root = GobFpath.cwd_append (fst (Fpath.split_base (Fpath.v Sys.argv.(0)))) in
-  let dist_dir = Fpath.(goblint_root // (Fpath.v "_build/default/gobview/dist")) in
-  let js_file = Fpath.(dist_dir / "main.js") in
   if gobview then (
-    if Sys.file_exists (Fpath.to_string js_file) then (
       let save_run = GobConfig.get_string "save_run" in
       let run_dir = Fpath.v(if save_run <> "" then save_run else "run") in
```

Comment:
`make view` copies things into `run/` but this output can be configured to go into other directories, where the files would be missing. Not sure what we should do about that.

Reply:
Good point. I think it does not really need to be configurable, but we should maybe choose a different name then for this folder.

Reply:
It might even be better if the compiled Gobview (of which there is just one) didn't go to the same directory as each Goblint marshalled output (of which there could be multiple with `save_run`). From `./run` it'd be possible to refer to `../gobview-dist/` or whatnot. But maybe modern web browsers/servers forbid such up-going relative path in some cases, not sure.

Reply:
In https://github.com/goblint/gobview/pull/33 I made a suggestion on how one can change the goblint-http server to serve from both directories (with priority on gobview-dist).